pkg/crc/api/handlers.go

```diff
@@ -61,16 +60,13 @@ func (h *Handler) Stop() string {
 }
 
 func (h *Handler) Start(args json.RawMessage) string {
-	var parsedArgs startArgs
-	var err error
+	var parsedArgs client.StartConfig
 	if args != nil {
-		parsedArgs, err = parseStartArgs(args)
-		if err != nil {
-			startErr := &client.StartResult{
+		if err := json.Unmarshal(args, &parsedArgs); err != nil {
+			return encodeStructToJSON(&client.StartResult{
 				Success: false,
 				Error:   fmt.Sprintf("Incorrect arguments given: %s", err.Error()),
```

Comment:
This is the code path that was tested as i mentioned in  https://github.com/code-ready/crc/pull/2342/commits/9e6e49b9f06c592d7716cde0e4a702a5248d2395#r634089624

Reply:
I added 2 tests for this: one with an invalid json, one with no body